src/main/java/tc/oc/pgm/events/PlayerPunishmentEvent.java

```diff
@@ -0,0 +1,63 @@
+package tc.oc.pgm.events;
```

Comment:
Keep all of these files, including events, in the `tc.oc.pgm.community` package.

So this would be `tc.oc.pgm.community.events`